lib/active_fedora/file.rb

```diff
@@ -134,6 +134,10 @@ def metadata
       @metadata ||= ActiveFedora::WithMetadata::MetadataNode.new(self)
     end
 
+    def checksum
+      ActiveFedora::Checksum.new(self)
```

Comment:
OK, well maybe we merge this and address any caching separately if anybody thinks that's important?